### ISSUE TITLE
Can load lbrynet on Linux Mint

### ISSUE THREAD

**Issue by rtevans01** [lbryio/lbry-sdk#3714]
Am I missing a detail somewher? I simply downloaded the file, extracted it, and then clicked on it but nothing happens. I made sure the box for "Allow executing file as program" was checked. Any help would be appreciated. Thanks.

**Comment by keikari** [comment]
What happens if you launch it from cli? 
`./lbrynet start`

**Comment by rtevans01** [comment]
> ./lbrynet start

> cd Downloads/zip
> ~/Downloads/zip$ ./lbrynet start
> 2022-11-26 10:07:28,652 INFO     lbry.extras.daemon.daemon:489: Starting LBRYNet Daemon
> 2022-11-26 10:07:28,652 INFO     lbry.extras.daemon.daemon:491: Platform: {
>   "processor": "x86_64",
>   "python_version": "3.7.15",
>   "platform": "Linux-5.15.0-53-generic-x86_64-with-debian-bullseye-sid",
>   "os_release": "5.15.0-53-generic",
>   "os_system": "Linux",
>   "lbrynet_version": "0.112.0",
>   "version": "0.112.0",
>   "build": "release",
>   "distro": {
>     "id": "linuxmint",
>     "version": "20.3",
>     "version_parts": {
>       "major": "20",
>       "minor": "3",
>       "build_number": ""
>     },
>     "like": "ubuntu",
>     "codename": "una"
>   },
>   "desktop": "X-Cinnamon"
> }
> 2022-11-26 10:07:28,653 ERROR    lbry.extras.daemon.daemon:503: RPC server failed to bind TCP localhost:5279
> 2022-11-26 10:07:28,654 INFO     lbry.extras.daemon.daemon:564: stopped api components
> 2022-11-26 10:07:28,654 INFO     lbry.extras.daemon.daemon:568: stopped api server
> 2022-11-26 10:07:28,654 INFO     lbry.extras.daemon.daemon:571: finished shutting down

Okay something happened. I guess it's running now? I see several lbrynet listings in my system monitor.

**Comment by keikari** [comment]
Do you possibly already have LBRY desktop app running? It also uses lbrynet daemon, and starts its own when started and if there isn't already running one.  Or then something else is using that same port.

**Comment by rtevans01** [comment]
Yes I had the LBRY client running. I didn't know it does the same thing. So I don't need to bother running lbrynet? 

I'm trying to get the LBRY addon in LibreELEC to work so I can download videos and support the network. I've tried just running the LBRY client on a separate PC on the same LAN and changing the API address in the addon to http://localhost:5279 but nothing changes.

**Comment by keikari** [comment]
Does LibreELEC have any debug tools? Are you getting any kind of error message?

One thing to try is to add this in your daemon_settings.yml:  
`allowed_origin: '*'`
https://lbry.tech/resources/daemon-settings

If you have something important in your wallet, make sure that the API port isn't reachable from outside of the trusted LAN when allowed_origin is any, since that line would allow anyone to run calls on your lbrynet, if you firewall isn't blocking their access.

**Comment by rtevans01** [comment]
Yes it can make logs. Here a [link](https://drive.proton.me/urls/S8D13H46D8#5UQDkJ8neLvF) to one I just made. Seem like there's a lot of stuff to sift through though.

I added `allowed_origin: '*'` to the yml config file but still no change. Switched to a different LAN for both my laptop and the RPI but no change.  Firewall was disabled the whole time. Here's the [output](https://privatebin.net/?0432a95da3de66bb=#2SCVKWXpHG3bYYZxBYanmFxpeJEjKLeDUFak8AHpfzNZ) from my terminal.

My wallet doesn't have any funds so I'm not worried about it.

I set the API in the YML config file to the port you suggested but nothing changed. Here's the [output](https://privatebin.net/?c2df63443330e2ca#FpzmKPGV5bzFoKmKjWkGeGVJD3o3d5sc8dtaaFKU1gsU).

**Comment by keikari** [comment]
In both lbrynet logs it seems to be listening in 127.0.0.1:5279, doesn't that mean it can't be reached outside of localhost.

**Comment by rtevans01** [comment]
I have no idea. 

I think I'll delete the modifications to the yml config file and hold off until things are more polished and easier to use. It would be nice if the LibreELEC addon offered login and download/upload capabilities on its own.

**Comment by jilv220** [comment]
I have the same issue. But I can connect through 127.0.0.1:5279 instead of localhost:5279.
